apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.V4Signer.cs

```diff
@@ -73,10 +73,24 @@ private struct SigningState
 
                 internal SigningState(RequestTemplate template, Options options, IBlobSigner blobSigner, IClock clock)
                 {
+                    _resourcePath = string.Empty;
+                    switch (options.UrlStyle)
```

Comment:
It's a pity that a switch expression doesn't really help here, given that you could now use it!

Reply:
Oh yes it does help, switch expressions work well with unnamed tuples. Take a look.